source/_integrations/systemmonitor.markdown

```diff
@@ -23,7 +23,7 @@ sensor:
   - platform: systemmonitor
     resources:
       - type: disk_use_percent
-        arg: /home
+        arg: /
```

Comment:
`/` is often incorrect as well, especially inside containers (which is the largest part of our user base). If we want something that works for the largest user group, I would suggest on changing it to `/config`.

```suggestion
        arg: /config
```

Reply:
`/` does work with OS since `/` is on the same data disk as `/config`. But for Supervised/Core in Container installation `/config` is better, and works in all cases, so I agree using `/config` is more likely what users want.